en/flight_controller/omnibus_f4_sd.md

```diff
@@ -0,0 +1,203 @@
+* MPU6000 Accel / Gyro
+* BMP280 Baro (not all boards have it mounted)
+* microSD (logging and parameters)
+* Futaba S.BUS and S.BUS2 / Spektrum DSM2 and DSMX / Graupner SUMD / PPM input / Yuneec ST24
+* OneShot PWM out (configurable)
+* built-in current sensor
+* built-in OSD chip (AB7456 via SPI, not supported yet)
+
+
+## Where to Buy
+
+The board is produced by different vendors, with some variations (e.g. with or without a barometer).
+
+> **Tip** PX4 is compatible with boards that support the Betaflight OMNIBUSF4SD target (if *OMNIBUSF4SD* is present on the product page the board should work with PX4).
+
+<span></span>
+> **Tip** Any Omnibus F4 labeled derivative (e.g. clone) should work as well. However, power distribution on these boards is of varying quality.
+
+These are the boards tested and known to work:
+- [Hobbywing XRotor Flight Controller F4](http://www.hobbywing.com/goods.php?id=590) 
+   > **Note** This board fits on top of the [Hobbywing XRotor Micro 40A 4in1 ESC](http://www.hobbywing.com/goods.php?id=588) without soldering. This ESC board also provides power for the Omnibus board.
+   
+   Purchase from:
+  * [Hobbywing XRotor F4 Flight Controller w/OSD](https://www.getfpv.com/hobbywing-xrotor-f4-flight-controller-w-osd.html) (getfpv)
+
+- Original Airbot Omnibus F4 SD
+
+  Purchase from:
+  * [Airbot (CN manufacturer)](https://store.myairbot.com/omnibusf4prov3.html)
+  * [Ready To Fly Quads (US reseller)](http://www.readytoflyquads.com/flip-32-f4-omnibus-v2-pro)
+
+
+Accessories include:
+* [ESP8266 WiFi Module](../telemetry/esp8266_wifi_module.md) for MAVLink telemetry.
+  You need to connect these pins: GND, RX, TX, VCC and CH-PD (CH-PD to 3.3V). The baud rate is 921600.
+
+  
+## Connectors
+
+Below are silkscreens for Omnibus F4 SD (V1), showing both top and bottom
+
+![Omnibus F4 SD v1 Silkscreen Top](../../assets/flight_controller/omnibus_f4_sd/silk-top.jpg)
+![Omnibus F4 SD v1 Silkscreen Bottom](../../assets/flight_controller/omnibus_f4_sd/silk-bottom.jpg)
+
+
+## Pinouts
+
+### Radio Control
+
+RC is connected to one of the following ports:
+- UART1
+- SBUS/PPM port (via inverter, internally goes to UART1)
+
+> **Note** Some Omnibus F4 boards have a jumper connecting either or both the MCU SBUS and PPM to a single pin header. Set your jumper or solder bridge to the appropriate MCU pin before use.
+
+
+### UARTs
+
+- UART6: GPS port
+  - TX: MCU pin PC6
+  - RX: MCU pin PC7
+
+  - Airbot Omnibus F4 SD Pinout is on Port J10 (TX6/RX6):
+
+  ![Omnibus F4 SD UART6](../../assets/flight_controller/omnibus_f4_sd/uart6.jpg)
+
+- UART4
+  - TX: MCU pin PA0
+  - RX: MCU pin PA1
+  - 57600 baud
+  - It can be configured via [SYS_COMPANION](../advanced_config/parameter_reference.md#SYS_COMPANION) parameter.
+
+  - Airbot Omnibus F4 SD Pinout:
+    - TX: RSSI pin
+    - RX: PWM out 5
+
+  ![Omnibus F4 SD UART4](../../assets/flight_controller/omnibus_f4_sd/uart4.jpg)
```

Comment:
I don't understand what I am supposed to be seeing in these images - perhaps label the bits of interest?